src/repository/projects.rs

```diff
@@ -54,9 +54,9 @@ impl ProjectRepository for PgProjectRepository {
     async fn get(&self, id: i32) -> Result<Vec<Project>, ProjectGetError> {
         let projects = sqlx::query_as::<_, Project>(
             "
-            SELECT project_id, main_document_id, owner, name
+            SELECT project_id, projcet_name, created_at, last_modified, owner_id_id
             FROM projects
-            WHERE projects.owner = $1
+            WHERE projects.owner_id = $1
```

Comment:
You haven't changed column name, there is no `owner_id` column

Reply:
more things changed now